.gitattributes

```diff
@@ -50,6 +50,9 @@ README.*   text eol=lf
 VERSION   text eol=lf
 
 # These files should have CRLF
+win_build_output/midl/brave/*/*/*/*/*/*/*/*.c   text eol=crlf
```

Comment:
This can be replaced by `win_build_output/midl/brave/**/*.c`:

```
mkdir play-with-gitattributes
cd play-with-gitattributes
git init
mkdir a
mkdir a\b
mkdir a\b\c
mkdir a\b\c\d
echo "hi" > a\b\c\d\e.txt
echo f > f.txt
git add -A
git commit -m "Add some files"
echo a/**/*.txt foo=bar > .gitattributes
git check-attr foo a\b\c\d\e.txt f.txt
```

gives:

```
"a\\b\\c\\d\\e.txt": foo: bar
f.txt: foo: unspecified
```